python/ray/experimental/state/api.py

```diff
@@ -200,23 +200,22 @@ def _make_http_get_request(
                 )
 
                 response.raise_for_status()
-            except Exception as e:
+            except requests.exceptions.RequestException as e:
                 err_str = f"Failed to make request to {self._address}{endpoint}. "
 
                 # Best-effort to give hints to users on potential reasons of connection
                 # failure.
-                if isinstance(e, requests.exceptions.ConnectionError):
-                    err_str += (
-                        "Failed to connect to API server. Please check the API server "
-                        "log for details. Make sure dependencies are installed with "
-                        "`pip install ray[default]`."
-                    )
+                err_str += (
```

Comment:
any reason why we remove ConnectionError?

Reply:
ConnectionError is a [derived class](https://requests.readthedocs.io/en/v3.0.0/_modules/requests/exceptions/) of that `requests.exceptions.RequestException` - so I am catching more requests related exceptions, which I feel should be handled similarly (i.e. a request to server failure)